crates/swc_ecma_codegen/tests/test262-min/e9682c37a1a959e1.js

```diff
@@ -1 +1 @@
-"\u{20BB7}\u{91CE}\u{5BB6}"
```

Comment:
Is this change intentional?

Reply:
Related https://github.com/swc-project/swc/pull/4037/#issuecomment-1068468119, I think we should avoid it, unicode parsing is slow